packages/nimble-components/src/rich-text/specs/README.md

```diff
@@ -200,6 +381,69 @@ _CSS Classes and CSS Custom Properties that affect the component_
 
 _Note_: This initial component design serves as a starting point for implementation, and it may undergo changes once the visual design is completed.
 
+#### User mention element:
+
+This element serves as a container for the mapping elements and additionally functions as a configuration element. It allows the transmission of
+arbitrary values that are specific to the `@mention` users within the rich text components. Currently, this component do not have any
+configuration properties/attributes for user mentions and plays the role of containing element for the user mapping elements.
+
+_Component Name_
+
+-   `nimble-rich-text-mention-users`
+
+_Content_
+
+-   One or more `nimble-mapping-mention-user` elements
+
+#### Mapping element (user mention):
+
+This mapping element is employed to establish a connection between the value displayed in the mapping view and the corresponding value stored within
+the markdown string. For instance, the username for an `@mention` is contained in the `text` attribute, which is used for display in the mention
+view, while the user ID is contained in the `key` attribute which is used to store within the markdown string.
+
+_Component Name_
+
+-   `nimble-mapping-mention-user`
+
+_Props/Attrs_
+
+-   `key`: string | number | boolean
+-   `text`: string
```

Comment:
I'd be a bit hesitant to use the attribute name since it is a built-in attribute in HTML unless we can convince ourselves it's highly valuable and we are implementing the exact same semantics the browser does.

Reply:
display-name or display-text could be good alternatives

Reply:
Updated to `display-name`